cgroup_top.py

```diff
@@ -41,10 +42,12 @@
     sys.exit(0)
 
 def cmd_exists(cmd):
-    return subprocess.call(["/bin/which",  cmd], stdout=subprocess.PIPE, stderr=subprocess.PIPE) == 0
+    return subprocess.call(["/usr/bin/which",  cmd], stdout=subprocess.PIPE, stderr=subprocess.PIPE) == 0
 
 HAS_LXC = cmd_exists('lxc-start')
 HAS_DOCKER = cmd_exists('docker')
+HAS_OPENVZ = cmd_exists('vzctl')
+regexp_ovz_container = re.compile('^/\d+$')
```

Comment:
I know (at least) 1 real world setup where this will break. This is when user ids are used as cgroups identifiers. I'd recommend to go with an additional lightweight check, maybe against a /proc file or equivalent?